api/src/controllers/db-doc.js

```diff
@@ -29,7 +29,21 @@ const isValidRequest = (method, docId, body) => {
   return true;
 };
 
-const requestError = res => {
+const isValidAttachmentRequest = (params, query) => {
+  if (params.attachmentId && !query.rev) {
+    return false;
+  }
+
+  return true;
+};
+
+const requestError = (res, status) => {
```

Comment:
Because there's no shared code between 403s and 404s maybe this should be two separate functions, eg: `notFoundError` and `permissionsError` or something?